### PR TITLE
use slug campain instead of selectedCampaignId

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
use slug campaign instead of selectedCampaignId=1233 in url

### CODE REVIEW

**Comment by nguyenhoaidanh** [comment]
@nguyenhuudungz pls review it. 
if it's ok. let's merge it into your branch.

File: src/pages/App.tsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ export default function App() {
                     <Route exact path="/referral" component={CreateReferral} />
                     <Route exact path="/discover" component={TrueSight} />
                     <Route exact path="/buy-crypto" component={BuyCrypto} />
-                    <Route exact path="/campaigns" component={Campaign} />
+                    <Route exact path={`${AppPaths.CAMPAIGN}/:slug?`} component={Campaign} />
```
Comment (nguyenhuudungz) [inline src/pages/App.tsx:192]: Why use `?` in here?

Comment (nguyenhoaidanh) [reply src/pages/App.tsx:192]: optional param

**Review by nguyenhuudungz — APPROVED** [review]
LGTM.

**Comment by nguyenhoaidanh** [comment]
QC test done, and it is not related any task, so I will merge it into main.   Is it ok, @nguyenhuudungz ?

**Comment by nguyenhuudungz** [comment]
> QC test done, and it is not related any task, so I will merge it into main.   Is it ok, @nguyenhuudungz ?

Yes.

**Pull request merged**